qutebrowser/commands/command.py

```diff
@@ -205,7 +205,16 @@ def get_arg_info(self, param):
 
     def get_pos_arg_info(self, pos):
         """Get an ArgInfo tuple for the given positional parameter."""
-        name = self.pos_args[pos][0]
+        try:
+            name = self.pos_args[pos][0]
+        except IndexError:
+            # Extra arguments could belong to a variadic positional
+            signature = inspect.signature(self.handler)
+            name = next((p.name for p in signature.parameters.values()
+                         if p.kind == inspect.Parameter.VAR_POSITIONAL), None)
+            if name is None:
+                raise
+
```

Comment:
How about a test for this in [`test_cmdutils`](https://github.com/The-Compiler/qutebrowser/blob/a24391d40d742ea2348da1fcc3bd5f058ba60dca/tests/unit/commands/test_cmdutils.py#L298)?